modules/cnv-virtctl-commands.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-using-the-cli-tools.adoc
+
+[id="cnv-virtctl-commands_{context}"]
+= Virtctl client commands
+
+The `virtctl` client is a command-line utility for managing {ProductName}
+resources. The following table contains the `virtctl` commands used throughout
+the {ProductName} documentation.
+
+.Virtctl client commands
+
+[width="100%",cols="42%,58%",options="header",]
+|===
+|Command |Description
+
+|`virtctl start <vm>` 
+|Start a virtual machine, creating a virtual machine instance.
+
+|`virtctl stop <vmi>` 
```

Comment:
`<vmi>` in this context should be `<vm>` -- the operation happens at the VM level. Same for restart